xbmc/pvr/windows/GUIWindowPVRSearch.cpp

```diff
@@ -65,6 +84,10 @@ CGUIWindowPVRSearchBase::CGUIWindowPVRSearchBase(bool bRadio, int id, const std:
 {
 }
 
+CGUIWindowPVRSearchBase::~CGUIWindowPVRSearchBase()
```

Comment:
Is there anything planned for here?

Reply:
If the compiler generates the dtor inline, or you define it yourself inline, then you would need to include "EpgSearchfilter.h" in the header, otherwise the compiler complains about the incomplete type when it tries to destruct the `std::unique_ptr<CPVREpgSearchFilter>` member which I have introduced with this PR.

<img width="310" alt="Screenshot 2019-03-09 at 23 45 56" src="https://user-images.githubusercontent.com/3226626/54078262-86426700-42c5-11e9-85d3-17d0b5989008.png">